ui/src/landing/wishTypeList/index.js

```diff
@@ -7,12 +7,12 @@ import Telescope from '../../assets/images/icn_To_See_Telescope_White_Inside_130
 import { WishTypeCard } from '../wishTypeCard'
 import './styles.scss'
 
-export const WishTypeList = ({ selectWishType, validFields, age }) => {
+export const WishTypeList = ({ selectWishType, validFields, age, name }) => {
   const { GO, MEET, BE, HAVE } = WishType
 
-  const {validAge} = validFields(age);
+  const {validAge, validName} = validFields(age, name);
   
-  if (validAge) {
+  if (validAge && validName) {
```

Comment:
I feel like we want to do a dirty check on the form before immediately showing an error. Right now the below is what I see loading up the app:  

<img width="985" alt="Screen Shot 2019-10-11 at 3 31 40 PM" src="https://user-images.githubusercontent.com/6162814/66680241-0f47c500-ec3e-11e9-85b8-245396511937.png">

Could you update this to check if the form has been touched yet before throwing an error. We probably want to remove the default age of `2` when you do that.

Reply:
Yup, I'll take care of that. A dirty check would be great for this